docs/src/appendix/cheatcodes/caller_address/prank.md

```diff
@@ -0,0 +1,10 @@
+# `prank`
+
+> `fn prank(target: CheatTarget, caller_address: ContractAddress, span: CheatSpan)`
+
+Changes the caller address for the given target and span.
+This change can be canceled with [`stop_prank`](./stop_prank.md).
+
+- `target` - instance of [`CheatTarget`](../cheat_target.md) specifying which contracts to prank
+- `caller_address` - caller address to be set
+- `span` - instance of [`CheatSpan`](../cheat_span.md) specifying the span of prank
```

Comment:
xD

Reply:
You suggested not to use duration, so :D

Reply:
What I meant is: if you are using duration to explain span, maybe duration is a better name after all :D . I think what we have now is ok.